pkgs/desktops/gnome-3/update.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, writeScript, python3, common-updater-scripts, coreutils, gnugrep, gnused }:
+{ packageName, attrPath ? packageName, versionPolicy ? "odd-unstable" }:
+
+let
+  python = python3.withPackages (p: [ p.requests ]);
+in writeScript "update-${packageName}" ''
```

Comment:
I do not really like that this creates a derivation per package updated but `maintainers/scripts/update.nix` is quite limited.